packages/hoppscotch-app/components/environments/Details.vue

```diff
@@ -130,136 +130,146 @@ import {
   setGlobalEnvVariables,
   updateEnvironment,
 } from "~/newstore/environments"
-import { useReadonlyStream } from "~/helpers/utils/composables"
+import {
+  useReadonlyStream,
+  useI18n,
+  useToast,
+} from "~/helpers/utils/composables"
+
+const t = useI18n()
+const toast = useToast()
+
+const props = withDefaults(
+  defineProps<{
+    show: boolean
+    action: string
+    editingEnvironmentIndex: number | string | null
```

Comment:
Correction. The only string possible (as per the old code) is "Global". Use TypeScript's String Literal Types for it.
Read: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types

`editingEnvironmentIndex: number | "Global" | null`